concepts/blocks/about.md

```diff
@@ -0,0 +1,59 @@
+# About
+
+A block is a set of statements and declarations grouped into one syntactic unit.
+Blocks are often, but not always, defined within curly braces.
+Blocks can be nested within functions and within other blocks.
+
+In the following example, `number` is defined within the `main` function.
```

Comment:
Do you mean to imply that a function itself constitutes a block? (As well as the signature)
If yes, then it may be best to explain that explicitly.
This would then make clearer the meaning of "inner block" as used in following paragraphs.

Reply:
Yeah, thanks, I've been thinking about whether to further distinguish blocks vs functions. For instance, two variables can have the same name in different blocks in the same function, but two labels cannot, but that really relates more to function scope. I can probably think a bit more objectively about how to put it with a little time to switch from creating mode to editing mode.

Reply:
For the "about" I added the bit about labels to explain the difference between function and block. I left that detail out of the introduction.